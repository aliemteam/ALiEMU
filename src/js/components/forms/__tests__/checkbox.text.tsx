import React from 'react';
import renderer from 'react-test-renderer';

import Checkbox from '../checkbox';

it('should render correctly', () => {
    const tree = renderer.create(
        <>
            <Checkbox label="Unchecked" />
            <Checkbox label="Checked" checked />
        </>,
    );
    expect(tree).toMatchSnapshot();
});
