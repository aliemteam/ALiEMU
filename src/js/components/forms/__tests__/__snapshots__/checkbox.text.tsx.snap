// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Array [
  <label
    className=""
  >
    <input
      checked={false}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      aria-checked={false}
      role="checkbox"
    />
    <span>
      Unchecked
    </span>
  </label>,
  <label
    className=""
  >
    <input
      checked={true}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      aria-checked={true}
      role="checkbox"
    >
      <i
        className="material-icons"
        style={
          Object {
            "color": undefined,
            "fontSize": 13,
          }
        }
      >
        check
      </i>
    </span>
    <span>
      Checked
    </span>
  </label>,
]
`;
